src/bin/typescript-demo-lib.ts

```diff
@@ -24,6 +27,50 @@ function main(argv = process.argv): number {
   const sum = nums.num1 + nums.num2;
   process.stdout.write(nums.num1 + ' + ' + nums.num2 + ' = ' + sum + '\n');
 
+  // Level tests.
```

Comment:
Don't use `console.log` here. It makes the test harder.

Use `process.stdout.write` like above.

Also I'd say put these into `src/lib/...` for each individual functionality. Provider functions like `level.ts` which exports a function that does the below, and same for the other ones.

Reply:
- Replaced each use of `consoel.log()` with `process.stdout.write()`.
- Separated each tests into it's own file inside `src/lib/`